source/DataAccess/Oracle/OracleReader.ts

```diff
@@ -81,12 +84,12 @@ export default class OracleReader implements IDataReader {
             });
 
             queryStream.on('error', (error: any) => {
-                this._logger.error(`Failed to execute: '${queryStatement}' - ${error.stack}`);
                 queryStream.destroy();
                 // Translate - ORA-00942: table or view does not exist
                 if (error.errorNum && error.errorNum === 942) {
-                    reject(new TableNotFoundException(queryStatement, error.message));
+                    reject(new TableNotFoundException(queryStatement, `Query execution unsuccessful. ${error.message}`));
```

Comment:
Updated the error to produce some thing more human friendly so the handler can output it as an informative warning

Reply:
👍